source/templates/home.html

```diff
@@ -17,7 +17,13 @@
     <p class="home-download-list">
       {{hb 'home.download_links'}}
     </p>
-    <p>{{link 'pages.changelog'}}</p>
+    <p>
+    {{#if i18n.home.custom_changelog}}
```

Comment:
I might change how this works long term for to make i18n links like this work better, but this is a good workaround for now, thank you for adding it.